src/EditProvider.ts

```diff
@@ -99,16 +100,84 @@ export class EditProvider
   }
 
   public static beautifyOptions(
-    filePath: string = vscode.workspace.rootPath
+    path: string = vscode.workspace.rootPath
   ): Promise<LanguageOptionValues> {
     try {
-      const explorer = cosmiconfig("unibeautify", {});
+      /**
+       * Use a different `.unibeautify` config when vscode config is set and
+       * define if the project config should extend the vscode config.
+       *
+       * @argument {unibeautify.defaultConfig} string path to a file
+       * @argument {unibeautify.extendConfig} boolean extend or not
```

Comment:
A comment should be unnecessary in this case so long as we have well written / readable code. Please remove 😄.

Now to look below and make sure we have readable code 😜 .